packages/gatsby/content/features/plugnplay.md

```diff
@@ -128,6 +128,7 @@ Many common frontend tools now support Plug'n'Play natively!
 | Babel | Starting from `resolve` 1.9 |
 | Create-React-App | Starting from 2.0+ |
 | Docusaurus | Starting from 2.0.0-beta.14 |
+| ESBuild | Starting from [0.15.0](https://github.com/evanw/esbuild/releases/tag/v0.15.0) |
```

Comment:
Probably should wait for 0.15.2, for https://github.com/evanw/esbuild/issues/2458 to be fixed